runbot_pylint/models/runbot_build.py

```diff
@@ -0,0 +1,217 @@
+# -*- encoding: utf-8 -*-
+##############################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 2011 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: moylop260@vauxoo.com
+############################################################################
+
+"""
+This module is used to create new fields in the inherited classes
+to work with pylint from runbot
+"""
+
+from openerp import models, api, fields
+import logging
+import os
+import stat
+from openerp.tools.safe_eval import safe_eval
+
+
+_logger = logging.getLogger(__name__)
+
+
+def get_depends(modules, addons_paths, depends=None):
+    """
+    Get recursive depends from addons_paths and modules list
+    """
```

Comment:
Could you add sphinx type `:param:`, `:type:` and `:return:` documentation for this function, it is a little confusing with the arguments and their types.